testing/test_integration_twisted.py

```diff
@@ -29,14 +29,6 @@ def fin():
     request.addfinalizer(fin)
     return server
 
-
-@pytest.inlineCallbacks
-def test_toggle(server):
```

Comment:
this is tested by test_toggle, and a bunch of times in the integration harness as well